src/f5_tts/model/mlx_modules.py

```diff
@@ -0,0 +1,512 @@
+from __future__ import annotations
```

Comment:
probably no separate mlx_modules.py in this repo,
leveraging existing modules.py is better for repo maintainability